Thank you ChatGPT.

I modified my test_tts.py as you recommended.

test_tts.py
import subprocess

VOICE = "/home/nickspi5/Chatty_AI/voices/en_US-amy-low/en_US-amy-low.onnx"
CONFIG = "/home/nickspi5/Chatty_AI/voices/en_US-amy-low/en_US-amy-low.json"
TEXT = "Hello Nick, this is a test of Piper text to speech on your Raspberry Pi 5"
OUTPUT = "output.wav"

# Run Piper
command = [
    "./piper/piper",
    "--model", VOICE,
    "--config", CONFIG,
    "--output_file", OUTPUT
]

subprocess.run(command, input=TEXT.encode("utf-8"))

I ran (chatty-venv) nickspi5@raspberrypi1:~/Chatty_AI $ python3 test_tts.py
terminate called after throwing an instance of 'nlohmann::json_abi_v3_11_2::detail::parse_error'
  what():  [json.exception.parse_error.101] parse error at line 1, column 1: syntax error while parsing value - invalid literal; last read: 'E'
(chatty-venv) nickspi5@raspberrypi1:~/Chatty_AI $ 


How can I fix this error?





